cabal.project

```diff
@@ -95,6 +95,14 @@ allow-newer:
   , inline-r:containers
   , inline-r:primitive
 
+-- Uncomment the following lines to make cardano-constitution:create-json-envelope buildable:True
```

Comment:
This is probably a silly question, but is there no way to tell cabal to only build this if the user turns on some flag?

Reply:
Good question, but I don't know how :man_shrugging: